### PR TITLE
fix: on-chain emotes definition & refactor of wearables

### PR DESCRIPTION
**Pull request by lorux0**
## What does this PR change?

Changes how on-chain emotes are defined through its urn. In addition a refactor was made to improve the separation with the wearables.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

File: Explorer/Assets/DCL/AvatarRendering/Wearables/Components/Wearable.cs [hunk 1]
```diff
@@ -43,6 +43,13 @@ public Wearable(StreamableLoadingResult<WearableDTO> dto)
             IsLoading = false;
         }
 
+        public bool IsOnChain()
+        {
+            IAvatarAttachment attachment = this;
+            var id = attachment.GetUrn().ToString();
+            return !id.StartsWith("urn:decentraland:off-chain:base-avatars:");
```
Comment (popuz) [inline Explorer/Assets/DCL/AvatarRendering/Wearables/Components/Wearable.cs:50]: I would inline in just one line or at lest merge 48 with 49

File: Explorer/Assets/DCL/AvatarRendering/Emotes/Emote.cs [hunk 2]
```diff
@@ -17,6 +17,13 @@ public class Emote : IEmote
 
         public bool IsLoading { get; set; } = true;
 
+        public bool IsOnChain()
+        {
+            IAvatarAttachment attachment = this;
+            var id = attachment.GetUrn().ToString();
+            return id.StartsWith("urn:") && !id.StartsWith("urn:decentraland:off-chain:");
```
Comment (popuz) [inline Explorer/Assets/DCL/AvatarRendering/Emotes/Emote.cs:24]: I would inline in just one line or at lest merge 22 with 23

**Review by popuz — APPROVED** [review]

File: Explorer/Assets/DCL/AvatarRendering/Emotes/Emote.cs [hunk 3]
```diff
@@ -17,6 +17,12 @@ public class Emote : IEmote
 
         public bool IsLoading { get; set; } = true;
 
+        public bool IsOnChain()
+        {
+            var id = ((IAvatarAttachment) this).GetUrn().ToString();
+            return id.StartsWith("urn:") && !id.StartsWith("urn:decentraland:off-chain:");
```
Comment (AlejandroAlvarezMelucciDCL) [inline Explorer/Assets/DCL/AvatarRendering/Emotes/Emote.cs:23]: Maybe I'm just tired, but wouldn't the second condition already cover the first one? We're comparing twice something we can do once, just like you did for `Wearable.cs`. What do you think?

Also, what do you think about turning the literals to constants?

**Review by AlejandroAlvarezMelucciDCL — APPROVED** [review]
LGTM just left a small suggestion :)

File: Explorer/Assets/DCL/AvatarRendering/Emotes/Emote.cs (hunk 3 above)
Comment (lorux0) [reply Explorer/Assets/DCL/AvatarRendering/Emotes/Emote.cs:23]: I belive its not the same, you can check the inputs in the unit tests

**Pull request merged**